tools/libboard/board_dc/board_dc.c

```diff
@@ -129,7 +142,8 @@ fpga_result parse_fw_ver(char *buf, char *fw_ver, size_t len)
 
 	errno = 0;
 	var = strtoul(buf, &endptr, 16);
-	if (!endptr || *endptr != '\0') {
+	if (var == 0 &&
```

Comment:
This should be..
`if (endptr != buf + strlen(buf)) {`

Reply:
if (endptr != buf + strlen(buf))  doesn't fix strtoul(), shows error message

Reply:
What are the errors?

Reply:
fixed, driver ends with new line char (\n).